src/core/layout/qgslayoutexporter.cpp

```diff
@@ -2190,3 +2239,52 @@ bool QgsLayoutExporter::saveImage( const QImage &image, const QString &imageFile
   }
   return w.write( image );
 }
+
+QString QgsLayoutExporter::getCreator()
+{
+  return QStringLiteral( "QGIS %1" ).arg( Qgis::version() );
+}
+
+void QgsLayoutExporter::setXmpMetadata( QPdfWriter *pdfWriter, QgsLayout *layout )
+{
+#if QT_VERSION >= QT_VERSION_CHECK(6, 8, 0)
+  QUuid uuid = pdfWriter->documentId();
+#else
+  QUuid uuid = QUuid::createUuid();
+#endif
+
+  // XMP metadata date format differs from PDF dictionary one
+  const QDateTime creationDateTime = layout->project()->metadata().creationDateTime();
+  const QByteArray creationDateMetadata = creationDateTime.toOffsetFromUtc( creationDateTime.offsetFromUtc() ).toString( Qt::ISODate ).toUtf8();
+
+  const QByteArray author = layout->project()->metadata().author().toUtf8();
+  const QByteArray creator = getCreator().toUtf8();
```

Comment:
XML escaping might be needed here (in case characters like &, <, > would be used). Also applies for pdfWriter->title()

Reply:
Right. Done [b8a1577](https://github.com/qgis/QGIS/pull/58511/commits/b8a1577cf1725a34b3aa40721cf582dd2fc73a6c)